src/components/buttons/ScrollUpButton/ScrollUpButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames/bind';
+import UpArrow from '@/public/svgs/upArrow.svg';
+import styles from './ScrollUpButton.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface ScrollUpButtonProps {
+  headerRef: React.RefObject<HTMLDivElement>;
```

Comment:
```suggestion
import { useState, useEffect, RefObject } from 'react';
import classNames from 'classnames/bind';
import UpArrow from '@/public/svgs/upArrow.svg';
import styles from './ScrollUpButton.module.scss';

const cn = classNames.bind(styles);

interface ScrollUpButtonProps {
  headerRef:RefObject<HTMLDivElement>;
```
이렇게 하면 불필요한 React 모듈 전체를 가져오는 것을 피할 수 있습니다!! :)

Reply:
네넵 최대한 import 문에 쓰도록 하겠습니다!!